deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpDeviceOperations.java

```diff
@@ -14,9 +14,12 @@
 
 public class AmqpDeviceOperations
 {
+    @SuppressWarnings("CanBeFinal") // Can be inherited and changed somewhere else besides the constructor
```

Comment:
Is it better to suppress at class level?

Reply:
We thought about this, however we have inconsistent protection levels and if we make changes to the class with private members we want the analyzer to produce meaningful warnings in the future.